XUtils.c

```diff
@@ -95,7 +95,21 @@ void* xReallocArrayZero(void* ptr, size_t prevmemb, size_t newmemb, size_t size)
 }
 
 inline bool String_contains_i(const char* s1, const char* s2) {
```

Comment:
I don't think this should be implemented as part of this function. Make the search logic it's own function as this may interfere with existing other callers.

Reply:
I moved the only other caller out of the way. That is the other commit :-)

Reply:
Ah, that other remark I had … ;-)